### DISCUSSION TITLE
Snap modifier for FLN mod

### DISCUSSION THREAD

**Discussion by Rhyzt** [Quaver/Quaver#4798] [Feature Request]
### Is your feature request related to a problem? Please describe.

Some maps have a low bpm, so fln mod doesn't really have that much density as i should wish

### Describe the solution you'd like

after activating the fln mod, you can select which snap you want the release to be (with the default in 1/4)

**Comment by EpiCreeper** [comment]
That could actually be good for multiple cases, some maps use 1/3 timing and adjusting the time for that could make fln more fun for that. I personally also like easy 1/2 release fln. It would probably mess with the ranking but it would be fun. This could also be useful for mapping.

**Comment by YagaoDirac** [comment]
Not only a auto snap mod is needed. A auto trim(against notes repeating too quickly) mod is also needed. Also a "turn notes into start/end of long notes" mod is needed.
